src/modules/home/components/images-carousel.tsx

```diff
@@ -0,0 +1,99 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Slider from 'react-slick';
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import { type AccommodationImage } from '../types/accommodation-image.type';
+import { NoDataAvailable } from '#/shared/components/no-data-response';
+
+const styles = {
+  prevButton: {
+    position: 'absolute',
+    top: '50%',
+    left: '-170px',
+    transform: 'translateY(-50%)',
+    color: '#fff',
+    minWidth: '50px',
+    height: '50px',
+    borderRadius: '50%',
+    boxShadow: '0 4px 10px rgba(0,0,0,0.3)',
+    zIndex: 10,
+    '&:hover': { backgroundColor: 'GrayText' },
+  },
+  nextButton: {
+    position: 'absolute',
+    top: '50%',
+    right: '-170px',
+    transform: 'translateY(-50%)',
+    color: '#fff',
+    minWidth: '50px',
+    height: '50px',
+    borderRadius: '50%',
+    boxShadow: '0 4px 10px rgba(0,0,0,0.3)',
+    zIndex: 10,
+    '&:hover': { backgroundColor: 'GrayText' },
+  },
+  container: {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: '90%',
+    height: '90%',
+    maxWidth: '850px',
+    p: 4,
+    borderRadius: 2,
+  },
+  image: {
+    width: '100%',
+    height: 'auto',
+    objectFit: 'cover',
+    borderRadius: 2,
+  },
+};
+
+function CustomPrevArrow({ onClick }: { onClick?: () => void }) {
```

Comment:
Let's create components/image-carousel folder. Inside, let's have image-carousel.tsx and navigation-buttons.tsx. Will be easier to read and component will be smaller I think

Reply:
done